pytorch_lightning/loggers/neptune.py

```diff
@@ -284,9 +538,7 @@ def name(self) -> str:
         Returns:
             The name of the experiment if not in offline mode else "offline-name".
         """
-        if self.offline_mode:
```

Comment:
Won't the `offline_mode` be supported anymore ?

Reply:
We still suuport `offline_mode`, but we don't have to check it since function always returns "NeptuneLogger".

The function comment is not up to date, but our product team is currently working on docstrings ;)